tools/interop_matrix/run_interop_matrix_tests.py

```diff
@@ -166,6 +175,24 @@ def find_test_cases(lang, runtime, release, suite_name):
                         '--server_host_override=(.*).sandbox.googleapis.com',
                         line)
                     server = m.group(1) if m else 'unknown_server'
+
+                    # If server_host arg is not None, replace the original 
+                    # server_host with the one provided or append to the end of
+                    # the command if server_host does not appear originally.
+                    if args.server_host:
+                      l_index = line.find('--server_host=')
```

Comment:
I think using regular expressions (`import re`) would lead to more readable code.

Reply:
I agree, but I didn't seem to get it to work. I'll try harder. :)

Reply:
Done.